docs/users_guide/ocean/test_groups/global_ocean.rst

```diff
@@ -305,14 +305,30 @@ ocean domain.
 A variant of this mesh has been used for low resolution simulations as part of
 the `E3SM v1 Cryosphere Campaign <https://e3sm.org/research/cryosphere-ocean/v1-cryosphere-ocean/>`_.
 
+.. _global_ocean_mesh_kuroshio:
+
+Kuroshio8to60 and Kuroshio12to60
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Kuroshio 8- and 12- to 60-km meshes (Kuroshio8to60 and Kuroshio12to60) are
+designed to explore dynamics of the Kuroshio Current.
+
+The meshes have 8 and 12 km resolution, respectively, in the eastern
+North Pacific, tapering to 60 km at mid latitudes, 30 km at the equator, and
+35 km in polar regions (the same as the EC30to60).
```

Comment:
@ytakano3, same here.  Please make any suggested changes here with the "Add a suggestion" button.

Reply:
@xylar Minor comment (change) added.